src/features/click-to-play.js

```diff
@@ -696,6 +946,72 @@ const config = {
                 }
             }
         }
+    },
+    'Google LLC': {
```

Comment:
I think we need to change the entity name here, to match the block list changes.
```suggestion
    'Youtube': {
```

Reply:
👍 ✅